mono/utils/mono-threads-coop.c

```diff
@@ -129,6 +129,9 @@ mono_threads_state_poll_with_info (MonoThreadInfo *info)
 
 	THREADS_SUSPEND_DEBUG ("FINISH SELF SUSPEND OF %p\n", mono_thread_info_get_tid (info));
 
+	/* Fast fail if wont_safept is set */
+	g_assert (!(info->thread_state & THREAD_SUSPEND_WONT_SAFEPT_MASK));
+
```

Comment:
Nit: More hard to read abbreviations. `WONT` and `SAFEPT` are in poor form imho.

Reply:
going to `s/wont_safept/no_safepoints/ig`